services/management/grpc/actions_server.go

```diff
@@ -260,38 +274,62 @@ func (s *actionsServer) StartMongoDBExplainAction(ctx context.Context, req *mana
 	}, nil
 }
 
-// StartPTSummaryAction starts pt-summary action.
+// StartPTSummaryAction starts pt-summary action. If the time since the last successful start of pt-summary action is lower than ptSummaryRefreshPeriod,
+// the response from the last action will be used. If the time is longer, the new pt-summary action will be called.
 //nolint:lll
-func (s *actionsServer) StartPTSummaryAction(ctx context.Context, req *managementpb.StartPTSummaryActionRequest) (*managementpb.StartPTSummaryActionResponse, error) {
-	agents, err := models.FindPMMAgentsRunningOnNode(s.db.Querier, req.NodeId)
+func (s *actionsServer) StartPTSummaryAction(ctx context.Context, psReq *managementpb.StartPTSummaryActionRequest) (*managementpb.StartPTSummaryActionResponse, error) {
+	// Gets current timestamp
+	timeNow := time.Now().Unix()
```

Comment:
In general, we try to add comments describing _why_ something is done, not _what_, unless the code in question is somewhat complicated. In that particular case, it doesn't look like this comment adds anything. We avoid 90% of those comments :)

![image](https://user-images.githubusercontent.com/11512/98701860-ca371a80-238a-11eb-84ee-121d5285cf5f.png)

Reply:
Pretty cat. Reminds me of my sister-in-law who is crazy about cats.

Reply:
You can say I'm crazy about cats too 😸

Reply:
Ok. I'll tell her to name her next cat AlekSi.